src/Sarif.Converters/ContrastSecurityConverter.cs

```diff
@@ -511,26 +492,24 @@ private Result ConstructFormsAuthenticationSSLResult(ContrastLogReader.Context c
             string path = properties[nameof(path)];
             string snippet = properties[nameof(snippet)];
 
-            return new Result
+            Result result = CreateResultCore(context);
+            result.Locations = new List<Location>()
```

Comment:
>() [](start = 49, length = 2)

super NIT but i should point this out since you cared enough about this in some places that you removed the redundant () :)

Reply:
Please do point out this sort of thing wherever you see it. I thought I got them all, will make another pass for "()".

---
In reply to: [274084528](https://github.com/Microsoft/sarif-sdk/pull/1421#discussion_r274084528) [](ancestors = 274084528)